tests/test_functional.py

```diff
@@ -120,3 +124,48 @@ def test_invalid_input(self, user, testapp):
             assert "Import proběhl úspěšně" not in res
         assert Record.query.count() == existing_records
         assert Batch.query.count() == existing_batches
+
+
+class TestDonorsOverview:
+
+    TEST_IMPORTS_DF = pandas.read_csv(
+        "tests/data/imports.csv", dtype={"RC": str, "PSC": str, "POJISTOVNA": str}
+    ).head(1000)
+    TEST_RC = sample(list(TEST_IMPORTS_DF.RC.unique()), 100)
```

Comment:
Přijde mi teda trochu odvážné předpokládat, že prvních 1 000 záznamů bude obsahovat 100 jedinečných rodných čísel. Ale jestli se to řadí podle pořadí v souboru a `head(1000)` vezme vždy ty samé řádky z _imports.csv_, pak si tím jisti být můžeme.